.storybook/util/flattenToken.js

```diff
@@ -0,0 +1,27 @@
+import { forEach, isObject } from 'lodash';
+
+export default function flatten(obj, themePrefix = '') {
```

Comment:
TODO: document and add tests

Given an object of an arbitrary depth, flatten the names using dashes (configure?), and add to any provided suffix. Append the suffix as you recurse.

When it reaches an `@`, treat that as the root name.